lib/generator.js

```diff
@@ -136,6 +152,20 @@ class Generator {
     });
   }
 
+  /**
+    * Check if the Registry Options are valid or not.
+    * 
+    * @public
```

Comment:
that should be privat, we do not want anyone to use that function

Reply:
Done @derberg